slack-post-message/README.md

```diff
@@ -0,0 +1,55 @@
+# slack-post-message
+
+slack-post-message adds a "Post a message" button in the slack shortcuts section. This can be used to post a message in multiple channels. The message will be posted in the channels selected, by the bot on behalf of the user. Only the users part of the *slack usergroups* given in the config file are permitted to use the bot. This app uses [Slack modals](https://api.slack.com/surfaces/modals) rather than dialogs
+
+![screenshot of a blank send message modal](./screenshot.png)
+
+## Configuration
+
+slack-post-message requires a configuration file, by default called `config.json` in the working
+directory. It must look like this:
+
+```json
+{
+  "signingSecret": "some_slack_signing_secret",
+  "accessToken": "xoxp-some-slack-access-token-these-are-very-long-and-start-with-xoxp",
+  "userGroups": ["UsergroupID1", "UsergroupID2"]
+}
+```
+`signingSecret`, `accessToken` are all values provided by Slack when creating and
+installing the app. Check out the [slack app creation guide][app-creation] for more details.
+
+For `userGroups`, from slack browser view find the ID(s) of the usergroups in the URL placeholder of the [browser](https://stackoverflow.com/questions/40940327/what-is-the-simplest-way-to-find-a-slack-team-id-and-a-channel-id)
+
+### Slack setup
+
+slack-post-message requires the following OAuth scopes:
+
+- `usergroups:read`
+- `channels:read`
+- `groups:read`
+- `chat:write`
+
+slack-post-message requires the following interactive components:
+
+- Callback ID: `write_message`
+- Recommended action name: "Post message"
+- The location required is 'Global'
+
+Set the request URL for the interactive component appropriately
```

Comment:
can we add that the URL is defined as `PATH_PREFIX+/webhook`?

Reply:
Yes, I will add it. It makes sense to be clear on the URL. Thanks!